web-wa-sqlite/README.md

```diff
@@ -127,6 +121,73 @@ const ExampleComponent = () => {
 }
 ```
 
+## Migrating the app
+
+ElectricSQL supports migrating the back-end while the app is running.
+Migrations are additive-only.
+For example, while running the app we may connect to the Postgres backend and add a column `other_value` to the `items` table:
+```shell
+docker exec -it -e PGPASSWORD=password local-stack-postgres_1-1  psql -h 127.0.0.1 -U postgres -d electric
+electric=# ALTER TABLE items ADD other_value TEXT;
+WARNING:  assigning automatic migration version id: 20230529120539_974
+ALTER TABLE
+```
+
+This migration will automatically be picked up by Electric and will be streamed to the application
+which will migrate its local SQLite database.
+
+Then, remains to migrate the code of our application to do something with the new column.
+To this end, first run the migration script from within the top-level directory of this app:
+
+```shell
+sh migrate.sh -f prisma/dump.sql -p prisma/schema.prisma
+Prisma schema loaded from prisma/schema.prisma
+Datasource "db": SQLite database "electric-tmp.db" at "file:electric-tmp.db"
+
+✔ Introspected 1 model and wrote it into prisma/schema.prisma in 18ms
+      
+Run prisma generate to generate Prisma Client.
+
+┌─────────────────────────────────────────────────────────┐
+│  Update available 4.8.1 -> 4.14.1                       │
+│  Run the following to update                            │
+│    npm i --save-dev prisma@latest                       │
+│    npm i @prisma/client@latest                          │
+└─────────────────────────────────────────────────────────┘
+Prisma schema loaded from prisma/schema.prisma
+
+✔ Generated Prisma Client (4.8.1 | library) to ./node_modules/@prisma/client in 32ms
+
+✔ Generated Zod Prisma Types to ./prisma/generated/models in 17ms
+```
+
+The migration script updated the Electric client to incorporate the new column `other_value` on the `items` table.
```

Comment:
> to incorporate the new column `other_value` on the `items` table.
Similarly to before, isn't it just modifying the type to include the new column?